### ISSUE TITLE
event discarded

### ISSUE THREAD

**Issue by hm3623851** [resmoio/kubernetes-event-exporter#184]
i see the log 'Event discarded as being older then maxEventAgeSeconds' in my loki and see client-side throttling with lots of 0 on my pod logs.
i add kubeQPS: 100 and kubeBurst: 500 to my config map and my maxEventAge is 300  and it works good about 24 hours after that again i see Event discarded as being older then maxEventAgeSeconds.

**Comment by hm3623851** [comment]
please help.

**Comment by Cloud-Shen** [comment]
> i see the log 'Event discarded as being older then maxEventAgeSeconds' in my loki and see client-side throttling with lots of 0 on my pod logs. i add kubeQPS: 100 and kubeBurst: 500 to my config map and my maxEventAge is 300 and it works good about 24 hours after that again i see Event discarded as being older then maxEventAgeSeconds.

same problem， have you solved ?

**Comment by hm3623851** [comment]
No, I haven't resolved it yet, but I have written a job to reset my
kube-event-exporter pods across the entire cluster every 24 hours.
